less/forms.less

```diff
@@ -0,0 +1,37 @@
+.contact {
+  width: 100%;
+}
+
+.contact-form {
+.flexy(flex, column, null, center, left);
+  width: 900px;
+  max-width: 100%;
+  padding: 2em 1em 0px 1em;
+  & input,
+  & select,
+  & textarea {
+    margin: 5px 0px;
```

Comment:
The `&` selector isn't needed here - nesting an element selector by itself already achieves the same effect.